CMakeLists.txt

```diff
@@ -562,12 +562,53 @@ set (gmic_qt_FORMS
 
 if(ENABLE_DYNAMIC_LINKING)
   set(CMAKE_SKIP_RPATH TRUE)
-  set(gmic_qt_LIBRARIES
-    ${gmic_qt_LIBRARIES}
-    "gmic"
+  # G'MIC-Qt needs visibility into the private symbols defined
+  # by the gmic.cpp plugin. However, this is only possible
+  # if the library is static OR if it's dynamic and built by
+  # a compiler that supports .so-style exports.
+  if (TARGET libgmicstatic OR MSVC OR NOT ENABLE_SYSTEM_GMIC)
+    set(gmic_qt_LIBRARIES
+      ${gmic_qt_LIBRARIES}
+      libgmicstatic
+    )
+  elseif(TARGET libgmic)
+    set(gmic_qt_LIBRARIES
+      ${gmic_qt_LIBRARIES}
+      libgmic
```

Comment:
Good catch, looks like it was not actually linking the exported targets before:

https://github.com/GreycLab/gmic/blob/d7d1182b8bfcbeae62da27891e28aa11f4a39c1b/resources/CMakeLists.txt#L164